cves/kernel/CVE-2015-8812.yml

```diff
@@ -55,7 +60,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: The vulnerability is a use-after-free vulnerability where the return value was improperly checked. *_send functions return a non-negative upon success and a negative number upon failure. By passing back the return value as-is in iwch_l2t_send and iwch_cxgb3_ofld_send, any functions that rely on 0 being success would interpret the return value as an error. For example, in send_connect (https://github.com/torvalds/linux/blob/c5dfb000b90442e52f735bef4c2fc3f67783eaeb/drivers/infiniband/hw/cxgb3/iwch_cm.c#L1970) which calls iwch_l2t_send, a non-zero return value will cause the *_release functions to be called. This is where the problem lies as all of these functions decrement the refcount on the associated structures which will cause them to be freed unexpectedly resulting in a use-after-free. 
```

Comment:
The example here is a bit much and makes the description hard to read, especially with the function names. Your explanation about how the functions return a value based on success/fail is good enough without the function names.

Reply:
I personally disagree about the function names as mentioning the function names helps potential readers follow the codepath and understand how the UAF came to be. However, I did remove the link to the source code as that did make it too verbose.